presto-main/pom.xml

```diff
@@ -366,10 +361,6 @@
             <artifactId>jjwt-impl</artifactId>
             <scope>runtime</scope>
         </dependency>
-        <dependency>
```

Comment:
Could you please confirm if we can eliminate this dependency? The error may be caused by removing it.